### ISSUE TITLE
Thumbnails

### ISSUE THREAD

**Issue by chrisvanbuskirk** [dropbox/dropbox-sdk-obj-c#174]
Anyway we can write a thumbnail for our files that would show in files app?

NSURL allows for:

@{NSURLThumbnailDictionaryKey: @{NSThumbnail1024x1024SizeKey: image}};

**Comment by greg-db** [comment]
No, the Dropbox SDK doesn't offer anything like that, but I'll pass this along as a feature request.
